sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -239,6 +248,8 @@ def get_out_data(self):
 
     def get_requester_nonce(self):
         """Return requesterNonce work order parameter."""
+        if not (self.params_obj["requesterNonce"]).strip():
```

Comment:
Good to add this check in set rather in get call.

Reply:
done